Header header

float64 frequency
string essid
string interface
string APaddr
uint8 status

uint8 STATUS_CONNECTED = 0
uint8 STATUS_DISCONNECTED = 1
uint8 STATUS_ERROR = 2